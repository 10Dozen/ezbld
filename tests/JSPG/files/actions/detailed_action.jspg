@ Detailed Action | dialog_right mypic.jpg | MyTag

*scene: CurrentScene
*icon: {img: my_picture.jpg}
*condition: "Foo.Bar"
**exec:
    let foo = bar()
    foo.update(100)
*goto: "SceneAnother"

Some line of description.