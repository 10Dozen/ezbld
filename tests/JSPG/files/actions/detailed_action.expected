[0]
cls = JSPGAction
name = Detailed Action
type = dialog_right
portrait = mypic.jpg
tag = MyTag
scene = CurrentScene
icon = {img: my_picture.jpg}
condition = "Foo.Bar"
exec =
    \tlet foo = bar()
    \tfoo.update(100)
goto = "SceneAnother"

desc = Some line of description.